activemq/datadog_checks/activemq/data/metrics.yaml

```diff
@@ -54,3 +54,86 @@ jmx_metrics:
         MemoryPercentUsage:
           alias: activemq.broker.memory_pct
           metric_type: gauge
+
+  - include:
+      domain: org.apache.activemq.artemis
+      attribute:
+        AddressMemoryUsage:
+          alias: activemq.artemis.address_memory_usage
+          metric_type: gauge
+        AddressMemoryUsagePercentage:
+          alias: activemq.artemis.address_memory_usage_pct
+          metric_type: gauge
+        MaxDiskUsage:
+          alias: activemq.artemis.max_disk_usage
+          metric_type: gauge
+        DiskStoreUsage:
+          alias: activemq.artemis.disk_store_usage_pct
+          metric_type: gauge
+        ConnectionCount:
+          alias: activemq.artemis.connection_count
+          metric_type: gauge
+        TotalConnectionCount:
+          alias: activemq.artemis.total_connection_count
+          metric_type: gauge
+        TotalMessageCount:
+          alias: activemq.artemis.total_message_count
+          metric_type: gauge
```

Comment:
should these be counters?

Reply:
In fact, `monotonic_count` - made several other updates.